src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/PersonalizeRankingResponseProcessor.java

```diff
@@ -41,7 +42,7 @@ public class PersonalizeRankingResponseProcessor extends AbstractProcessor imple
 
     private static final Logger logger = LogManager.getLogger(PersonalizeRankingResponseProcessor.class);
 
-    public static final String TYPE = "personalize_ranking";
+    public static final String TYPE = "personalized_search_ranking";
```

Comment:
Do we want to change the class name as well? src/main/java/org/opensearch/search/relevance/transformer/personalizeintelligentranking/PersonalizeRankingResponseProcessor.java

Also there are multiple names now: `personalizeintelligentranking`, `PersonalizeRanking`, `personalized_search_ranking`. Can we make it consistent?

Reply:
Agree. We will address in next iterations to make the naming more consistent.